src/cpp/include/nodegui/QtGui/QFontDatabase/qfontdatabase_wrap.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <napi.h>
+#include <stdlib.h>
+
+#include <QFontDatabase>
+
+#include "core/Component/component_macro.h"
+
+class QFontDatabaseWrap : public Napi::ObjectWrap<QFontDatabaseWrap> {
+ private:
+  std::unique_ptr<QFontDatabase> instance;
+
+ public:
+  static Napi::FunctionReference constructor;
+  static Napi::Object init(Napi::Env env, Napi::Object exports);
+  QFontDatabaseWrap(const Napi::CallbackInfo& info);
+  QFontDatabase* getInternalInstance();
+  // Wrapped methods
+  Napi::Value bold(const Napi::CallbackInfo& info);
+  Napi::Value italic(const Napi::CallbackInfo& info);
+  Napi::Value weight(const Napi::CallbackInfo& info);
+  Napi::Value addApplicationFont(const Napi::CallbackInfo& info);
```

Comment:
Few changes here. addApplicationFont and removeApplicationFont are static methods. So it should be moved from here. You can checkout out  `nodegui/src/cpp/include/nodegui/QtGui/QApplication/qapplication_wrap.h`  for reference on how to do that.
There is a separate `StaticQApplicationWrapMethods` namespace.

Reply:
oh, my bad, i will fix it